SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -411,12 +411,8 @@ void CTurbSSTSolver::BC_HeatFlux_Wall(CGeometry *geometry, CSolver **solver_cont
         /*--- Set wall values ---*/
         su2double density = solver_container[FLOW_SOL]->GetNodes()->GetDensity(iPoint);
         su2double laminar_viscosity = solver_container[FLOW_SOL]->GetNodes()->GetLaminarViscosity(iPoint);
+        su2double WallShearStress = GetWallShearStress(val_marker, iVertex);
```

Comment:
The method needs to be called on the flow solver (`solver_container[FLOW_SOL]->`).

Reply:
Fixed. Actually I was just doing a debug and I found that there was a problem.
Thanks for reporting it!